include/fastdds/rtps/participant/RTPSParticipant.h

```diff
@@ -21,12 +21,14 @@
 
 #include <cstdlib>
 #include <memory>
+
 #include <fastrtps/fastrtps_dll.h>
 #include <fastdds/rtps/common/Guid.h>
 #include <fastdds/rtps/attributes/RTPSParticipantAttributes.h>
+#include <fastdds/rtps/builtin/data/ContentFilterProperty.hpp>
+#include <fastdds/statistics/IListeners.hpp>
 #include <fastrtps/qos/ReaderQos.h>
 #include <fastrtps/qos/WriterQos.h>
```

Comment:
I suggest that we take advantage of this PR to upgrade these two header files to the ones in the `fastdds` include folder

```suggestion
#include <fastdds/dds/publisher/qos/WriterQos.h>
#include <fastdds/dds/subscriber/qos/ReaderQos.h>
```

Reply:
As discussed offline. Changing this could break API for users not following include-what-you-use, as they could be including just `RTPSParticipant.h` and using `eprosima::fastrtps::ReaderQos` on their code.